.github/workflows/deployment-template.yml

```diff
@@ -51,14 +59,63 @@ jobs:
           OPENID_LOGOUT_URL: ${{ secrets.openid_logout_url }}
           GA_TRACKING_ID: ${{ secrets.ga-tracking-id }}
 
-      - name: Step 2 - Configure AWS Credentials
+      - name: Step 2 - Replace Tokens for logstash configuration
+        if: ${{inputs.deploy-env == 'preview' }}
```

Comment:
nit: same as above, add one space on the left as well `if: ${{ inputs.deploy-env == 'preview' }}`